### ISSUE TITLE
Add support for chapter-level notes

### ISSUE THREAD

**Issue by tobias-klein** [tobias-klein/ezra-project#94]

**Comment by zhuiks** [comment]
Isn't that already implemented?

**Comment by tobias-klein** [comment]
No. What we have today is verse-level notes and book-level notes. But not notes that apply for individual chapters. I am not sure whether we should implement this anytime soon ...
